modules/ROOT/pages/conf-examples/office-integration.adoc

```diff
@@ -0,0 +1,124 @@
+
+[source,plaintext]
+----
+127.0.0.1 ocis.owncloud.test
+127.0.0.1 traefik.owncloud.test
+127.0.0.1 collabora.owncloud.test
+127.0.0.1 onlyoffice.owncloud.test
+127.0.0.1 wopiserver.owncloud.test
+----
+
+For the clients, make sure that the addresses can be resolved to the server(s) running this setup.
+
+== Installation
+
+Installing the Docker stacks only requires a few steps. Take care to use the relevant version. Select a base folder of choice for the target location. Note that the examples use the latest stable version: {compose_tab_2}.
+
+. Get the deployment example:
++
+--
+Follow the xref:deployment/wopi/wopi.html#wopi-coniguration-examples[WOPI Coniguration Examples,window=_blank] to download the example files.
+
+After all resources have been downloaded, change into the directory:
+
+[source,bash]
+----
+cd ocis_wopi
+----
+--
+
+== Configuration
+
+The configuration uses a docker-compose file with an environment variable file for ease of handling. This docker-compose file defines which additional containers are needed and how the containers are configured.
+
+[start=2]
+. As first task, the xref:conf-examples/ext-files/list_file_1.adoc[environment definition file,window=_blank] file needs to be adapted. This file defines many basic settings in one location. Open it in a text editor and adjust it according to your needs.
++
+--
+For some hints on what to take into consideration, check the notes listed here:
+
+* If you are installing Infinite Scale on a server and Traefik will obtain valid certificates for you, remove `INSECURE=true` or set it to `false`.
+
+* If you want to use the Traefik dashboard, set `TRAEFIK_DASHBOARD` to `true` (default is `false` and therefore not active). If you activate it, you must set a domain for the Traefik dashboard in `TRAEFIK_DOMAIN=`, for example `TRAEFIK_DOMAIN=traefik.owncloud.test`.
+
+* The Traefik dashboard is secured by Basic Auth. Default credentials are the user `admin` with the password `admin`. To set your own credentials, generate a htpasswd (e.g. by using an https://htpasswdgenerator.de/[online tool] or a CLI tool).
+
+* Traefik will issue certificates with LetsEncrypt. Therefore you must set an email address in `TRAEFIK_ACME_MAIL=`.
+
+* By default Infinite Scale will be started with the latest version. If you want to run a specific version of Infinite Scale, set the version to `OCIS_DOCKER_TAG=`. Available versions can be found on https://hub.docker.com/r/owncloud/ocis/tags?page=1&ordering=last_updated[Docker Hub].
+
+* Set your domain for the Infinite Scale web frontend in `OCIS_DOMAIN=`, for example `OCIS_DOMAIN=ocis.owncloud.test`.
+
+* Set the initial admin user password in `ADMIN_PASSWORD=`. It defaults to `admin`.
+
+* By default, the CS3Org WOPI server will be started with version `v9.3.0`. If you want to start a different or more recent version, you can set the `WOPISERVER_DOCKER_TAG=`. Available versions can be found on {wopiserver_url}[Docker Hub].
```

Comment:
I would change that. The referenced example files will get regular updates. I would prefer "the CS3Org WOPI server will be started with the version from the examples, which might not be the latest stable released version.

Reply:
Fixed by @EParzefall